python/mxnet/optimizer.py

```diff
@@ -154,18 +154,18 @@ def create_state(self, index, weight):
         """
 
     def update(self, index, weight, grad, state):
-        """Update the weight given the corresponding gradient and state.
+        """Update the parameters given the corresponding gradients and state.
```

Comment:
Is this updating a single parameter tensor or multiple? If a single one let's just say: "Updates the given parameter using the corresponding gradient and state."